orca-pipelinetemplate/src/main/java/com/netflix/spinnaker/orca/pipelinetemplate/PipelineTemplatePipelinePreprocessor.java

```diff
@@ -66,18 +68,18 @@ public PipelineTemplatePipelinePreprocessor(ObjectMapper pipelineTemplateObjectM
 
     TemplateConfiguration templateConfiguration = request.getConfig();
 
-    // TODO find all templates via request.config.pipeline.template.source
-    // list needs to be FIFO, where the first template is the root
     List<PipelineTemplate> templates = templateLoader.load(templateConfiguration.getPipeline().getTemplate());
 
     PipelineTemplate template = TemplateMerge.merge(templates);
 
     GraphMutator graphMutator = new GraphMutator(templateConfiguration, renderer, registry);
     graphMutator.mutate(template);
 
-    // TODO final validation & marshal to pipeline json
+    // TODO validation
 
-    return pipeline;
+    ExecutionGenerator executionGenerator = new V1SchemaExecutionGenerator();
```

Comment:
Does the v1 here refer to orca's execution engine? Why do we want to run this stuff in v1?